python/ray/serve/_private/deployment_state.py

```diff
@@ -474,6 +475,19 @@ def start(self, deployment_info: DeploymentInfo) -> ReplicaSchedulingRequest:
         }
         actor_options.update(deployment_info.replica_config.ray_actor_options)
 
+        # A replica's default `max_concurrency` value may prevent it from
+        # respecting the configured `max_ongoing_requests`. To avoid this
+        # unintentional behavior, use `max_ongoing_requests` to override
+        # the Actor's `max_concurrency` if it is not explicitly set.
+        if (
+            "max_concurrency" not in actor_options
```

Comment:
`ray_actor_options` is limited to a structured schema `RayActorOptionsSchema`, which doesn't allow `max_concurrency`, so I think you don't need to check for `max_concurrency`

Reply:
yeah this was trying to future proof